relacy/context.hpp

```diff
@@ -30,9 +30,9 @@
 #include "stdlib/condition_variable.hpp"
 #include "stdlib/mutex.hpp"
 
-#include "random_scheduler.hpp"
-#include "full_search_scheduler.hpp"
-#include "context_bound_scheduler.hpp"
+#include "schedulers/random_scheduler.hpp"

```

Comment:
Commits like this need some explanation of the motivation.
Scheedulers structure changed. Ok, good. But why? Can we not do this change? What is the problem this commit solves?